packages/firestore/src/local/local_store.ts

```diff
@@ -642,6 +615,191 @@ export class LocalStore {
       });
   }
 
+  /**
+   * Applies the documents from a bundle to the "ground-state" (remote)
+   * documents.
+   *
+   * LocalDocuments are re-calculated if there are remaining mutations in the
+   * queue.
+   */
+  applyBundledDocuments(
```

Comment:
I'd prefer if we kept referring to the feature as "Bundle". `applyBundleDocuments` makes it clear that the documents are from a Firestore Bundle, whereas `applyBundledDocuments` just suggests that the documents were somehow bundled (possibly in a generic fashion).